browser/ui/color/brave_color_mixer.cc

```diff
@@ -633,6 +642,19 @@ void AddBraveOmniboxLightThemeColorMixer(ui::ColorProvider* provider,
                                          const ui::ColorProviderKey& key) {
   ui::ColorMixer& mixer = provider->AddMixer();
 
+  mixer[kColorBraveOmniboxResultViewSeparator] = {
+      leo::GetColor(leo::Color::kColorDividerSubtle, leo::Theme::kLight)};
+
+  // Apply bravified color when there is no custom theme.
+  if (key.custom_theme) {
+    mixer[kColorBraveOmniboxResultViewSeparator] = {
+        PickColorContrastingToOmniboxResultsBackground(
+            key, mixer,
+            leo::GetColor(leo::Color::kColorDividerSubtle, leo::Theme::kLight),
+            leo::GetColor(leo::Color::kColorDividerSubtle, leo::Theme::kDark))};
+    return;
+  }
+
```

Comment:
I think it'd be better to move the comment below to avoid confusion.

```suggestion
  if (key.custom_theme) {
    mixer[kColorBraveOmniboxResultViewSeparator] = {
        PickColorContrastingToOmniboxResultsBackground(
            key, mixer,
            leo::GetColor(leo::Color::kColorDividerSubtle, leo::Theme::kLight),
            leo::GetColor(leo::Color::kColorDividerSubtle, leo::Theme::kDark))};
    return;
  }

  // Bravified colors used only when there is no custom theme.
```

Reply:
Done.